awx/ui/client/src/templates/job_templates/factories/md-5-setup.factory.js

```diff
@@ -26,5 +27,3 @@ export default
             };
         };
     }
-
-md5Setup.$inject = [   'md5'   ];
```

Comment:
can probably remove some md5 libs now

Reply:
That'd be nice.  I dunno how to do that 🤷‍♂️